orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTaskSpec.groovy

```diff
@@ -182,23 +182,24 @@ class FindImageFromClusterTaskSpec extends Specification {
     ['foo-x86_64-201603232351']                                     | ['foo-x86_64-201603232351'] as Set
   }
 
-  @Unroll
+//  @Unroll
```

Comment:
isn't this `@Unroll` needed here to get all the test cases in the `where` clause?

Reply:
Oops, ya. Well it doesn't stop the tests from running it just runs them back-to-back. 

Re-enabled this in a subsequent PR.